src/components/ClassCard/ClassCard.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./ClassCard.css";
+import { IoRemoveCircle } from "react-icons/io5";
+
+export default function ClassCard({
+  id,
+  name,
+  duration,
+  intervalTime,
+  onRemoveClassClick,
+}) {
+  return (
+    <section className="ClassCard">
+      <h2>{name}</h2>
+      <div className="ClassCard__input">
+        <p>Duration</p>
```

Comment:
Could this an `h3`.

Reply:
It looks more prominent now with an h3, but I think I will keep it. Tks!

Reply:
I was thinking semantically. Not in style or layout ;-)